scripts/monitor/rano_monitor/utils.py

```diff
@@ -37,7 +37,7 @@ def get_hash(filepath: str):
 def run_editor(t1c, flair, t2, t1, seg, label, cmd=REVIEW_COMMAND):
```

Comment:
If now command path is passing explicitly, do you need to keep a default value for `cmd` here? it may be error-prone if you don't fix a call somewhere

Reply:
Completely agree. I'm removing the default value